build/shared-runtime.version

```diff
@@ -1 +1 @@
-1.2.0-beta-001304-00
\ No newline at end of file
+1.1.1;release/1.1.0,2.0.0-beta-001747-00;master
```

Comment:
In theory, the 2.0.0 runtime should be bundled with the CLI installer. This file is only for installing runtimes **not** included with the CLI. @pranavkm how to we ensure the package version of Microsoft.NETCore.App on our mirror lines up with the CLI's version?